definitions.h

```diff
@@ -439,6 +608,18 @@ void DefaultVertShader(Vertex & vertOut, Attributes & attrOut, const Vertex & ve
     attrOut = vertAttr;
 }
 
+// My vertex shader
+void transformShader(Vertex & vertOut, Attributes & attrOut, const Vertex & vertIn, const Attributes & attrIn, const Attributes & uniforms)
+ {
+    Matrix transform(uniforms.matrix * vertIn);
+    //transform.print();
+    vertOut = vertIn;
```

Comment:
I did this same thing, I couldn't really figure out why I couldn't find a more elegant solution to this but if you look up how Matthew Brown did his it helped me understand how to improve this shader.